### PR TITLE
Center Logo

### PR DESCRIPTION
**Pull request by jjasghar**
I _think_ this centers the logo.

### CODE REVIEW

**Review by russellb — CHANGES_REQUESTED** [review]
Doesn't look like it worked, preview is here: https://github.com/instructlab/.github/tree/jjasghar-patch-1/profile

**Comment by russellb** [comment]
It does look like it's possible with HTML, but it's seriously frowned upon in terms of markdown best practices to inline HTML.

I don't have a strong feeling about whether this is important enough to do something naughty in MD.

**Comment by jjasghar** [comment]
It seems someone has figured this out: https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d#center-alignment-example

File: profile/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Welcome to the Lab 🔬
 
-<center>
+<p align="center">
 ![image](https://github.com/instructlab/.github/blob/main/assets/logo.png)
```
Comment (russellb) [inline profile/README.md:4]: You have to convert this to ``

Comment (jjasghar) [reply profile/README.md:4]: Ah yep!

**Comment by jjasghar** [comment]
Yay there we go.

**Comment by russellb** [comment]
Yay! I'm willing to look the other way on inlining the HTML. :-D

Can you squash this down to 1 commit though?

**Comment by jjasghar** [comment]
Done.

**Review by russellb — APPROVED** [review]

**Pull request merged**